app/models/binary/explore/PrecomputedExplorer.scala

```diff
@@ -0,0 +1,64 @@
+package models.binary.explore
+import com.scalableminds.util.geometry.{BoundingBox, Vec3Double, Vec3Int}
+import com.scalableminds.util.tools.Fox
+import com.scalableminds.webknossos.datastore.dataformats.MagLocator
+import com.scalableminds.webknossos.datastore.dataformats.precomputed.{
+  PrecomputedDataLayer,
+  PrecomputedLayer,
+  PrecomputedSegmentationLayer
+}
+import com.scalableminds.webknossos.datastore.datareaders.AxisOrder
+import com.scalableminds.webknossos.datastore.datareaders.precomputed.{PrecomputedHeader, PrecomputedScale}
+import com.scalableminds.webknossos.datastore.models.datasource.{Category, ElementClass}
+
+import java.nio.file.Path
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class PrecomputedExplorer extends RemoteLayerExplorer {
+  override def name: String = "Neuroglancer Precomputed"
+
+  override def explore(remotePath: Path, credentialId: Option[String]): Fox[List[(PrecomputedLayer, Vec3Double)]] =
+    for {
+      infoPath <- Fox.successful(remotePath.resolve(PrecomputedHeader.METADATA_PATH))
+      precomputedHeader <- parseJsonFromPath[PrecomputedHeader](infoPath) ?~> s"Failed to read Precomputed metadata at $infoPath"
+      layerAndVoxelSize <- layerFromPrecomputedHeader(precomputedHeader, remotePath, credentialId)
+    } yield List(layerAndVoxelSize)
+
+  private def layerFromPrecomputedHeader(precomputedHeader: PrecomputedHeader,
+                                         remotePath: Path,
+                                         credentialId: Option[String]): Fox[(PrecomputedLayer, Vec3Double)] =
+    for {
+      name <- guessNameFromPath(remotePath)
+      boundingBox <- BoundingBox.fromSize(precomputedHeader.scales.head.size).toFox
+      elementClass: ElementClass.Value <- elementClassFromPrecomputedDataType(precomputedHeader.data_type) ?~> "Unknown data type"
+      smallestResolution = precomputedHeader.scales.head.resolution
+      voxelSize <- Vec3Int.fromList(smallestResolution.toList).toFox
+      mags: Seq[MagLocator] <- Fox.serialCombined(precomputedHeader.scales)(
```

Comment:
```suggestion
      mags: List[MagLocator] <- Fox.serialCombined(precomputedHeader.scales)(
```
(then you don’t need the `.toList`s below)